src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+}
+
+.navbar {
+  padding-left: 50px;
+  padding-right: 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+}
+
+.nav {
+  line-height: 14px;
+  text-transform: uppercase;
+  color: #000;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+}
+
+.nav__link {
+  display: block;
+  margin-left: 20px;
```

Comment:
Margin has to make distance between to neighbor elements (`<a class="nav__link"> ` and `<a class="nav__link">`  are not neighbors).
So, remove margin to `<li>`